src/strategy/stratgies.test.ts

```diff
@@ -1,26 +1,15 @@
-import { beforeEach, afterEach, expect, it } from 'vitest';
+import { expect, it } from 'vitest';
 
 import { parser } from '@shaderfrog/glsl-parser';
 import { generate } from '@shaderfrog/glsl-parser';
 
 import { applyStrategy, StrategyType } from '.';
-import * as graphModule from '../graph/graph';
 import { makeExpression } from '../util/ast';
 
 import { SourceNode } from '../graph/code-nodes';
 import preprocess from '@shaderfrog/glsl-parser/preprocessor';
-
-// let orig: any;
-// beforeEach(() => {
-//   orig = graphModule.mangleName;
-//   // Terrible hack. in the real world, strategies are applied after mangling
-//   // @ts-ignore
-//   graphModule.mangleName = (name: string) => name;
-// });
-// afterEach(() => {
-//   // @ts-ignore
-//   graphModule.mangleName = orig;
-// });
```

Comment:
switching to vitest removed the ability to monkeypatch like this, which is what sent me down the rabbit hole of trying not to care about mangling in strategies